pkg/newlog/newlogd-init.sh

```diff
@@ -1,29 +1,43 @@
 #!/bin/sh
 
+echo "$(date -Ins -u) starting newlogd"
 /usr/bin/newlogd &
-mkdir -p /run/watchdog/pid
-touch /run/watchdog/pid/newlogd.pid
+NEWLOGD_PID=$!
+echo "$(date -Ins -u) newlogd is starting..., pid $NEWLOGD_PID"
```

Comment:
NIT: here and other invocations of `data` - for readability it's always better to use full form of arguments in the scripts
```
--iso-8601=ns --utc
```

Reply:
Good point, modified. Another interesting thing here is that busybox's date from alpine 3.13 do not support `-Ins`, only `-Is` so I was forced to add `coreutils` package.
```
BusyBox v1.32.1 () multi-call binary.

Usage: date [OPTIONS] [+FMT] [TIME]

Display time (using +FMT), or set time

	[-s,--set] TIME	Set time to TIME
	-u,--utc	Work in UTC (don't convert to local time)
	-R,--rfc-2822	Output RFC-2822 compliant date string
	-I[SPEC]	Output ISO-8601 compliant date string
			SPEC='date' (default) for date only,
			'hours', 'minutes', or 'seconds' for date and
			time to the indicated precision
	-r,--reference FILE	Display last modification time of FILE
	-d,--date TIME	Display TIME, not 'now'
	-D FMT		Use FMT (strptime format) for -d TIME conversion

Recognized TIME formats:
	hh:mm[:ss]
	[YYYY.]MM.DD-hh:mm[:ss]
	YYYY-MM-DD hh:mm[:ss]
	[[[[[YY]YY]MM]DD]hh]mm[.ss]
	'date TIME' form accepts MMDDhhmm[[YY]YY][.ss] instead
```

Reply:
>  forced to add coreutils package

Yes, I ran into this earlier too. Adding coreutils is the right way to solve that, I think